requirements.d/all.txt

```diff
@@ -1,7 +1,7 @@
 # packages always needed
 dnspython
 netaddr
-django>=1.7.1, <1.8
+django>=1.8.7, <1.9
```

Comment:
Is there a special reason to require >= .7 ? pip will install the latest one anyway and some linux dists might have only e.g. 1.8.6.